libraries/core-react/src/components/Dialog/Dialog.tokens.ts

```diff
@@ -62,4 +63,9 @@ export const dialog: DialogToken = {
       minHeight: '48px',
     },
   },
+  modes: {
+    compact: {
+      height: shape._modes.compact.straight.minHeight,
```

Comment:
Suggest removing this for now as `height` is not used in the component and we also don't know how the "compact" dialog should look like.